src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2091,7 +2146,8 @@
     text: >
```

Comment:
When should we use ">"? When should we use "|"?

Reply:
They are almost the same except a small difference.

> If you want the linebreaks to be preserved as \n in the string (for instance, embedded markdown with paragraphs), use |
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-over-multiple-lines

Reply:
very deep insight